### ISSUE TITLE
Node 6, for..of seems to break wallaby

### ISSUE THREAD

**Issue by mvestergaard** [wallabyjs/public#585]
It seems that this bit of code is breaking wallaby:
```javascript
for (let [type, registration] of this.components) {
  result.push(registration);
}
```

`this.components` is a Map.

I get a pretty non-descript error saying that there's an error in the instrumented file at line 22.
Looking at the instrumented file at that line number:

```javascript
19 for (let [
20               type,
21               registration
22        ] of $_$w(4, 11), this.components) {
23   $_$w(4, 12), result.push(registration);
24 }
```

It appears the instrumentation does something wrong here, `$_$w(4, 11),` after the `of` keyword is probably what is breaking it.

**Comment by ArtemGovorov** [comment]
Interesting issue, thanks for the report.
What's interesting, it works ok with `for-in`, but not with `for-of`. Also [babel is ok with the sequence expression in `for-of`](https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Ces2015-loose%2Creact%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3&experimental=true&loose=true&spec=true&code=for%20(let%20%5Btype%2C%20registration%5D%20of%20c()%2C%5B%5D)%20%7B%0A%7D).
Will be fixing it shortly.

**Comment by ArtemGovorov** [comment]
Fixed the issue in the latest wallaby core v1.0.224. The core should be updated automatically, to force the update, please [follow these instructions](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).
